### ISSUE TITLE
Error on account/channel pages on peertube.social

### ISSUE THREAD

**Issue by Nutomic** [Chocobozzz/PeerTube#1722]
Hi, I am currently having an issue with peertube.social and I have no idea how to fix it. There are no error logs, 

The problem itself is pretty easy to reproduce: 
1. go to https://peertube.social
2. go to any profile page or channel page, like https://peertube.social/accounts/root/videos

This will show the following error. The problem is not 100% reproducible, sometimes it works randomly, but always breaks again if you keep visiting different profiles.

![Screenshot_20190319_211642](https://user-images.githubusercontent.com/1044450/54639024-b1c70d80-4a8c-11e9-8096-c5cdf8348fc8.png)

I already checked the Peertube logs, and there is absolutely nothing.

I did a migration from the "standard" Peertube install to a new server with Docker 2 days ago, so this might be related. However, I dont think that anything went wrong during the migration, because nothing else is broken on the instance, and as I mentioned, the profiles are accessible sometimes. Maybe this could be a problem with the Docker installation?

@Chocobozzz Feel free to contact me on Mastodon or Matrix to debug this. 

os version: Ubuntu 18.04.2 LTS
docker version: 18.09.2
docker-compose version: 1.17.1
docker-compose.yaml: https://gist.github.com/Nutomic/7a6e7a3ec568d573d243f71f7df7d540

**Comment by Booteille** [comment]
I get "Account not found" error when trying to go on any of the account of your instance except with root.

**Comment by Nutomic** [comment]
You're right, the root account is working. But I'm not sure how much this means, because previously accounts were also broken, and then were displayed correctly again.

**Comment by Chocobozzz** [comment]
It seems the API sends a 404 (example: https://peertube.social/api/v1/accounts/crickxson).

Please paste a row of a local account joined with the actor table (remove sensitive data)

**Comment by Nutomic** [comment]
Can you give me an sql statement for that?

**Comment by Nutomic** [comment]
Okay the problem should be fixed now. For some reason, the database was broken and strings didnt match any more, so statements like `SELECT * FROM actors WHERE name='myuser'` would fail, even if the user existed. I fixed it by exporting the database, creating a new db and importing the data. In the process I also had to delete some duplicate data that was generated in the mean time. I dont think the issue had anything to do with Peertube.

Thanks to @Chocobozzz for helping me to debug this :)
